turbinia/client.py

```diff
@@ -350,13 +350,15 @@ class TurbiniaServer(object):
 
   Attributes:
     task_manager (TaskManager): An object to manage turbinia tasks.
+    jobs_blacklist (list): Jobs we will exclude from running
```

Comment:
List of what? Strings? Class names? Instances? Please add this to the docstring.